rollup.config.js

```diff
@@ -0,0 +1,16 @@
+import babel from 'rollup-plugin-babel';
+import pkg from './package.json';
+
+export default [	
+	{
```

Comment:
Looks like this file uses tabs rather than spaces!

Reply:
I know, weird! in VSCode it appears as spaces:
![image](https://user-images.githubusercontent.com/45576380/64201332-7498e100-ce5c-11e9-9499-9b045e56fa17.png)